wiki/Game_modifier/Perfect/id.md

```diff
@@ -0,0 +1,54 @@
+---
+tags:
+  - perfect
+  - mod
+  - game modifier
+  - PF
+---
+
+# Perfect (mod)
+
+![Ikon mod PF](/wiki/shared/mods/PF.png "Ikon mod Perfect (PF)")
+
+*Untuk melihat daftar keseluruhan dari semua mod, kunjungi: [Game modifier](/wiki/Game_modifier)*\
+*Kunjungi juga: [Sudden Death (mod)](/wiki/Game_modifier/Sudden_Death)*
+
+## Tentang
+
+- Singkatan: PF
+- Tipe: Peningkat Kesulitan
+- Pengali Skor: 1.00x
+- Tombol shortcut default: `S` `S` atau `Shift`+`S`
+- Caption: `SS atau gagal.`
+- Mode-mode permainan yang cocok: ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+- Variasi: [Sudden Death](/wiki/Game_modifier/Sudden_Death)
+
+## Deskripsi
+
+Mod **Perfect** adalah sebuah [game modifier](/wiki/Game_modifier) yang bertujuan untuk membantu pemain yang mengejar skor SS dengan secara otomatis menggagalkan dan mengulangi [beatmap](/wiki/Beatmap) ketika pemain meleset mengenai sebuah note atau mendapatkan skor di [hit object](/wiki/Hit_object) yang bukan 300.
```

Comment:
```suggestion
Mod **Perfect** adalah sebuah [game modifier](/wiki/Game_modifier) yang bertujuan untuk membantu pemain yang mengejar skor SS dengan secara otomatis menggagalkan dan mengulangi [beatmap](/wiki/Beatmap) ketika pemain meleset mengenai sebuah not atau mendapatkan skor yang bukan 300 di [hit object](/wiki/Hit_object).
```

Reply:
reverting the sentence instead.

Reply:
wasn't there a talk where if there's only one "note" it should just stay as "note", and if there's "notes" (plural one) that's when you make it "not-not"?

Reply:
both is fine

Reply:
yeah it depends on context but i'm with FAMoss here in thinking that the Indonesian `not` (instead of the English `note) would fit this sentence better

Reply:
roger